modules/ROOT/pages/mule-deployment-model.adoc

```diff
@@ -4,13 +4,13 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: anypoint studio, esb, deploy, architecture
 
-A Mule Project for deployment is a *Mule Application*. A Mule runtime instance supports multiple applications running independently within the same Mule container. This allows additional advantages such as:
+A Mule project for deployment is a *Mule Application*. A Mule runtime instance supports multiple applications running independently within the same Mule container. This allows additional advantages such as:
 
-* Running multiple applications side-by-side
-* Clear boundaries for operations on what a Mule application is
-* Mule can monitor your applications and will reload configuration changes automatically
-* Applications can depend on different library versions, even if they would conflict before
-* Multiple versions of an application can exist side by side
+* Running independently multiple applications side-by-side. You can include the same name spaces within different applications and they neither collide nor share information.
```

Comment:
I am still not sure what the first sentence means. Does it mean "Run unrelated mule applications within the same Mule container"?

For the second sentence, I suggest "You can include the same namespaces within different applications without any conflicts."

Reply:
I changed it back to something different. _Running independently multiple applications side-by-side. If you have to build an application with some complexity, you can divide it by creating several apps with their specific logic and then deploy all the apps within the same Mule container._ but it was not reflected. You might need to click on *Files changes* in the GitHub page to see the real update.